fastlane_core/spec/configuration_spec.rb

```diff
@@ -85,113 +82,14 @@
         it "calls custom conflict handler when conflict happens between two options" do
           conflicting_options = [
             FastlaneCore::ConfigItem.new(key: :foo,
-                                         short_option: "-f",
                                          conflicting_options: [:bar, :oof],
                                          conflict_block: proc do |value|
-                                           UI.user_error!("You can't use option '#{value.short_option}' along with '-f'")
+                                           UI.user_error!("You can't use option '#{value.key}' along with 'foo'")
                                          end),
-            FastlaneCore::ConfigItem.new(key: :bar,
-                                         short_option: "-b"),
-            FastlaneCore::ConfigItem.new(key: :oof,
-                                         short_option: "-o",
-                                         conflict_block: proc do |value|
-                                           UI.user_error!("You can't use option '#{value.short_option}' along with '-o'")
-                                         end)
-          ]
-
-          values = {
-              foo: "",
-              bar: ""
-          }
-
-          expect do
-            FastlaneCore::Configuration.create(conflicting_options, values)
-          end.to raise_error "You can't use option '-b' along with '-f'"
-        end
-
-        it "raises an error for unresolved conflict between options" do
```

Comment:
Those tests are duplicated.

I submitted this PR as 2 commits: https://github.com/fastlane/fastlane/pull/4292/commits